internal/gengapic/testdata/custom_op_init.want

```diff
@@ -74,6 +77,12 @@ func NewRESTClient(ctx context.Context, opts ...option.ClientOption) (*Client, e
 	}
 	c.setGoogleClientInfo()
 
+	opC, err := NewFooOperationRESTClient(ctx, opts...)
```

Comment:
I wonder if we should be passing the httpClient to this method so they share the same base TokenSource under the hood. Thoughts?

Reply:
Great idea! I will do that.

Reply:
Done!